dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/client/LibertyWebServiceClientInInterceptor.java

```diff
@@ -105,13 +105,22 @@ public void handleMessage(Message message) throws Fault {
         }
         
 
-        // Enable or disable schema validation as long as property is non-null
+        // As long as property is non-null:
+        // Enable enhanced schema validation if true, or disable it along with default validation if false 
         if ( enableSchemaValidation != null) {
-            message.put("schema-validation-enabled", (boolean) enableSchemaValidation);
-
-            if (debug) {
-                Tr.debug(tc, "Set schema-validation-enabled to " + (boolean) enableSchemaValidation);
-
+            if ((boolean) enableSchemaValidation == true) {
+                // enable Schema Validation 
+                message.put("schema-validation-enabled", (boolean) enableSchemaValidation);
```

Comment:
Would it be better to use primitive value true, rather than enableSchemaValidation to be consistent with the line message.put("schema-validation-enabled", false);